### PR TITLE
add responseObject to error.userInfo

### PR DESCRIPTION
**Pull request by rayshih**

### CODE REVIEW

File: RACAFNetworking/AFHTTPSessionManager+RACSupport.m [hunk 1]
```diff
@@ -33,7 +33,12 @@ - (RACSignal *)rac_POST:(NSString *)path parameters:(id)parameters constructingB
 		
 		NSURLSessionDataTask *task = [self dataTaskWithRequest:request completionHandler:^(NSURLResponse *response, id responseObject, NSError *error) {
 			if (error) {
-				[subscriber sendError:error];
+        NSMutableDictionary *userInfo = [error.userInfo mutableCopy];
+        if (responseObject) {
+          userInfo[@"responseObject"] = responseObject;
```
Comment (CodaFi) [inline RACAFNetworking/AFHTTPSessionManager+RACSupport.m]: This needs to be formatted with tabs.

File: RACAFNetworking/AFHTTPSessionManager+RACSupport.m [hunk 2]
```diff
@@ -69,7 +74,12 @@ - (RACSignal *)rac_requestPath:(NSString *)path parameters:(id)parameters method
...
```
Comment (CodaFi) [inline RACAFNetworking/AFHTTPSessionManager+RACSupport.m]: This needs to be a constant string exposed in the header.  Perhaps

```
NSString *const RACAFNResponseObjectErrorKey;
```

File: RACAFNetworking/AFHTTPSessionManager+RACSupport.m (hunk 1 above)
Comment (rayshih) [inline RACAFNetworking/AFHTTPSessionManager+RACSupport.m]: OK I'll do it later

**Comment by CodaFi** [comment]
There we go.  Thanks for the pull request!  :sparkles: 

:boat:

**Comment by rayshih** [comment]
You're welcome! :)

**Pull request merged**